include/openthread.h

```diff
@@ -716,6 +716,56 @@ ThreadError otGetPendingDataset(otOperationalDataset *aDataset);
 ThreadError otSetPendingDataset(otOperationalDataset *aDataset);
 
 /**
+ * This function sends MGMT_ACTIVE_GET.
+ *
+ * @param[in]  aTlvTypes  A pointer to the TLV Types.
+ * @param[in]  aSize      The size of TLV Types.
+ *
+ * @retval kThreadError_None         Successfully send the meshcop dataset command.
+ * @retval kThreadError_Fail         Fail to send the meshcop dataset command.
```

Comment:
I'm not sure `kThreadError_Fail` is really possible.  However, I know `kThreadError_NoBufs` is a reason why the message may not be sent.